packages/ffe-cards/less/card-group.less

```diff
@@ -0,0 +1,58 @@
+@import 'common-card-styling';
+
+.ffe-card-group {
+    .common-card-styling();
+    border: none;
+    box-shadow: none;
+
+    &--box-shadow {
+        box-shadow: 0 1px 4px 0 var(--ffe-v-cards-common-card-box-shadow-color);
+    }
+
+    .ffe-card-base,
+    .ffe-icon-card,
+    .ffe-text-card {
+        box-shadow: none;
+        padding: var(--ffe-spacing-sm);
+        margin-bottom: 0;
+        background-color: transparent;
+
+        &:not(:last-child) {
+            border-radius: 0;
+            border-bottom: 1px solid var(--ffe-farge-lysvarmgraa);
+        }
+    }
+
+    .ffe-accordion-item {
+        box-shadow: none;
+        margin-bottom: 0;
+        background-color: transparent;
+    }
+
+    .ffe-accordion:not(:first-child) .ffe-accordion-item {
```

Comment:
Alt her og nedover er forslag hvis vi vil tilrettelegge for at teamene bruker accordion. Virker ikke hvis man vil ha noe på bunn, men det synes jeg ikke var så realistisk. Kan fjerne hvis det ble for spesifikt